### PR TITLE
init: Setup Dependencies, Project Structure, and README

### PR DESCRIPTION
**Pull request by sambit-sankalp**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+- Git Workflow
+
+```bash
+## Step 1: Fork Repository
+## Step 2: Git Set Up & Download
+# Clone the repo
+$ git clone https://github.com/<User-Name>/<Repo-Name>.git
+# Add upstream remote
+$ git remote add upstream https://github.com/dscnitrourkela/<Repo-Name>.git
```
Comment (nayakastha) [inline CONTRIBUTING.md:25]: add correct repo name instead of <repo-name>

**Review by nayakastha — CHANGES_REQUESTED** [review]

**Review by nayakastha — APPROVED** [review]

**Review by riteshsp2000 — APPROVED** [review]

**Comment by riteshsp2000** [comment]
Hey @sambit-sankalp You haven't added the following
- Github issues template
- Github PR template
- Github action for codeql-analysis

**Comment by CIPHERTron** [comment]
Heyy @sambit-sankalp have you made sure that ES Lint and Prettier are working correctly?

**Comment by CIPHERTron** [comment]
> Heyy @sambit-sankalp have you made sure that ES Lint and Prettier are working correctly?

nvm

**Review by CIPHERTron — APPROVED** [review]

**Pull request merged**